### PR TITLE
docs: update readme for cc3

### PR DESCRIPTION
**Pull request by soc1c**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,12 +59,12 @@ If you already have Rust installed, make sure you're using the latest version by
...
 ```
 
-### Install "Kusama" Canary Network
+### Install "Kusama CC-3" Canary Network
 
 Build Kusama by cloning this repository and running the following commands from the root directory of the repo:
 
 ```bash
-git checkout v0.6
+git checkout master
```
Comment (joepetrowski) [inline README.md]: Master may break w/r/t the CC3 releases, especially as experimental stuff gets merged in. I would just add a note to find and check out the latest `v0.7.*` tag, as all those should stay compatible with CC3.

Comment (soc1c) [reply README.md]: Why don't we just have a `v0.7` branch then? If master breaks stuff, we would need a release branch anyways?

Comment (joepetrowski) [reply README.md]: Yeah, fair enough

**Comment by soc1c** [comment]
Created a v0.7 release branch. cc @gavofyork @s3krit

**Pull request merged**